code/game/gamemodes/changeling/powers/mutations.dm

```diff
@@ -329,7 +329,7 @@
 	desc = "A tough, hard covering of black chitin with transparent chitin in front."
 	icon_state = "lingarmorhelmet"
 	flags = BLOCKHAIR | NODROP
-	armor = list(melee = 40, bullet = 15, laser = 30,energy = 10, bomb = 5, bio = 2, rad = 0)
+	armor = list(melee = 60, bullet = 30, laser = 60, energy = 30, bomb = 10, bio = 4, rad = 0)

```

Comment:
Uh, not exactly correct:

https://github.com/tgstation/-tg-station/blob/master/code/modules/clothing/suits/armor.dm#L18
https://github.com/tgstation/-tg-station/blob/master/code/modules/clothing/head/helmet.dm#L7